tests/pytests/functional/modules/test_test.py

```diff
@@ -0,0 +1,9 @@
+def test_deprecation_warnings_ignore(salt_call_cli):
+    """
+    Test to ensure when env variable PYTHONWARNINGS=ignore
+    is set that we do not add warning to output.
+    """
+    ret = salt_call_cli.run(
+        "--local", "test.deprecation_warning", env={"PYTHONWARNINGS": "ignore"}
+    )
```

Comment:
Can we also not set the environment variable to assert the deprecation warning is shown? Just to "counter" test.

Could be in the same test case.